etl-driver/src/main/java/com/wipro/ats/bdre/im/etl/api/StageLoad.java

```diff
@@ -61,50 +57,30 @@ public void execute(String[] params) {
         //Getting base table info
         String baseTableName = baseTable;
         String baseDbName = baseDb;
-      //  String baseTableDdl = getBaseTable().getDdl();
 
         processStageLoad(stageViewDbName, stageViewName, baseDbName, baseTableName,instanceExecId, minId, maxId,processId);
     }
 
     //Read the partition keys from hive table
     //input - database name, table Name
-    //output - partition column names as comma seperated;
+    //output - partition column names as comma seperated
 
 
     private void processStageLoad(String stageDbName, String viewName, String baseDbName, String baseTableName, String instanceExecId, String minBatchId, String maxBatchId,String stageLoadProcessId) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Method has 8 parameters, which is greater than 7 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS00107)